react/components/ErrorPage/messages/index.ts

```diff
@@ -0,0 +1,34 @@
+import bg from './bg.json'
+import de from './de.json'
+import en from './en.json'
+import es from './es.json'
+import fr from './fr.json'
+import it from './it.json'
+import ja from './ja.json'
+import ko from './ko.json'
+import nl from './nl.json'
+import pt from './pt.json'
+import ro from './ro.json'
+import th from './th.json'
+
+export const messages: Record<string, Record<string, string>> = {
+  'en-US': en,
+  'es-AR': es,
+  'fr-FR': fr,
+  'pt-BR': pt,
+  'ja-JP': ja,
+  'ko-KR': ko,
+  'it-IT': it,
+  'nl-NL': nl,
+  'ro-RO': ro,
+  'bg-BG': bg,
+  'th-TH': th,
+  'de-DE': de,
+}
+
+export function getMessages(locale: string) {
+  const localeMessages = (locale && messages[locale]) || {}
+  const fallbackMessages = messages['en-US']
```

Comment:
This is key and it answers @iago1501's question :heart:

Reply:
If it's because of the fallback we could simply use the available languages in vtex.store as an example of languages that we should take care in stores, for example: https://github.com/vtex-apps/store/tree/master/messages

or talking about admin, the ones that we have in admin-pages: https://github.com/vtex-apps/admin-pages/tree/master/messages

My main concern is that using the message's builder is one of our patterns when talking about translations, we have the messages folder and all translations are inside it, with this approach, talking about scaling, if any other component needs translation, we would have to add a messages folder for the component instead of center in a single place and this logic would be replied 🤔

Reply:
I see your point @iago1501, but given that this application didn't have translations until this day while being around for many years, I wouldn't be concerned about adopting a pattern that we might not need by the end of the day, and that we know it's the entry point for one of the most frequent errors we experience in the Admin given that the messages service itself didn't scale as we expected. 

For this same reason, it occurs to me that we should rather not rely on the messages service nor its builder, specially on a mission critical component such as this application, when we could otherwise organize translations where needed (the error component)  and skip all complexity that using the messages builder involves (triggering the messages service lifecycle just to fetch some strings stored elsewhere while it could just to be packaged with the application itself?).

It occurs to me that using the messages service could break the error component itself quite frequently.